patches/minecraft/net/minecraft/world/entity/boss/enderdragon/EnderDragon.java.patch

```diff
@@ -8,10 +8,10 @@
 +   }
 +
 +   @Override
-+   public void m_20234_(int p_20235_) {
-+      super.m_20234_(p_20235_);
++   public void m_20234_(int id) {
```

Comment:
Why is this changing from a srg name for the parameter to an explicitly set one? Is it just that we are patching in this entire method?

Reply:
Yes, it is because we are patching the entire method, and otherwise it would be polluted with the parchment name of `pId`, so I just went for `id`. I started to put this in the exc file, but it would be the only method in that file still in srg since it's an override, and I realized at that point that it should not have an exc.